.github/workflows/python-app.yml

```diff
@@ -5,9 +5,15 @@ name: Python build
 
 on:
   push:
-    branches: [master]
+    branches:
+      - main
+      # Temporary inclusion of master branch during default branch name transition
+      - master
   pull_request:
-    branches: [master]
+    branches:
+      - main
+      # Temporary inclusion of master branch during default branch name transition
+      - master
```

Comment:
@gwaybio @d33bs  I've made preliminary changes to allow the tests for this PR to run against both `master` and `main` branches. However, I think the best way to complete the transition would be for you to create a new `main` branch now and for me to change the merge target of this PR. Then after the merge to main is completed, changing the default branch. 

Does that sound like a good plan to you two?

Reply:
@kenibrewer great thoughts! Given your changes here I propose we merge as-is with `master` as the target (and remaining in the actions spec as a branch reference) then afterwards rename `master` to `main` referencing [GitHub's guidance on renaming branches](https://github.com/github/renaming#renaming-existing-branches). After the rename at some point a small PR could be issued to remove `master` references within for GitHub Actions or elsewhere. Thoughts?

Either way, doing this may incur confusion for in-flight PR's so we should be sure to notify those with open or draft state PR's about the change just in case anything unusual happens. Once you merge I can take responsibility for renaming the branch and sending along notifications to the open or draft PR's (granted this all sounds good).

CC @gwaybio

Reply:
@d33bs I like your plan a lot better than my suggestion. Let's proceed with that approach.